### PR TITLE
Misc. Improvements

### PR DESCRIPTION
**Pull request by yongla**
## Summary of Changes
- Always show Monday to Friday initially on `WeekView`
- Fixed `WeekView`'s nested scrolling on course detail page
- Moved some code into `ScheduleUtils` class
- Ran code formatter

### CODE REVIEW

File: app/src/main/java/com/example/peterchu/watplanner/ui/NestedWeekViewScrollView.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        final int action = ev.getAction();
+        switch (action) {
+            case MotionEvent.ACTION_DOWN:
+                super.onTouchEvent(ev);
+                break;
+
+            case MotionEvent.ACTION_MOVE:
+                // First view within the container of scrollview is the WeekView card
+                LinearLayout container = (LinearLayout) getChildAt(0);
+                View view = container.getChildAt(4);
```
Comment (timothytong) [inline app/src/main/java/com/example/peterchu/watplanner/ui/NestedWeekViewScrollView.java:39]: whats the 5th view in the calendar?

Comment (yongla) [reply app/src/main/java/com/example/peterchu/watplanner/ui/NestedWeekViewScrollView.java:39]: 5th view in the calendar? `View view = container.getChildAt(4);` is what gets the `LinearLayout` encompassing the `WeekView`. `container` is the container of the `ScrollView`

Comment (timothytong) [reply app/src/main/java/com/example/peterchu/watplanner/ui/NestedWeekViewScrollView.java:39]: oh okay

**Comment by timothytong** [comment]
LGTM

**Review by PeterChu2 — APPROVED** [review]
LGTM! 👍

**Pull request merged**